### ISSUE TITLE
How to set hyperparameters

### ISSUE THREAD

**Issue by xiaoqin1998** [HuCaoFighting/Swin-Unet#61]
I would like to ask everyone, except the batch size is 24, and the popular SGD optimizer with momentum 0.9 and weight decay 1e-4. What are the other superparameter settings? The results of my own training model are much worse.

**Comment by maralzar** [comment]
We had the same problem please check learning size and also imagenet pretrained model which load properly. Note the learning rate written in the Git , which varies by default.

**Comment by syhkqa** [comment]
解决了吗
